activity.py

```diff
@@ -8,52 +8,38 @@
 import sugargame.canvas
 from sugar3.activity import activity
 from sugar3.graphics.toolbarbox import ToolbarBox
-from sugar3.activity.widgets import ActivityToolbarButton
+from sugar3.activity.widgets import ActivityToolbarButton, StopButton
 from sugar3.graphics.radiotoolbutton import RadioToolButton
 from sugar3.graphics.toolbutton import ToolButton
-from sugar3.activity.widgets import StopButton
 from gettext import gettext as _
 import main
 import pygame
 
 
 class Activity(activity.Activity):
-
     def __init__(self, handle):
         activity.Activity.__init__(self, handle)
         self.max_participants = 1
         self.sound = True
         self.game = main.Flappy()
         self.build_toolbar()
-        self.game.canvas = self._pygamecanvas = \
-            sugargame.canvas.PygameCanvas(self,
-                                          main=self.game.run,
-                                          modules=[pygame.display, pygame.font,
-                                                   pygame.mixer])
-        self.set_canvas(self._pygamecanvas)
-        self._pygamecanvas.grab_focus()
+        self.game.canvas = sugargame.canvas.PygameCanvas(self, main=self.game.run, modules=[pygame.display, pygame.font, pygame.mixer])
```

Comment:
Line does not comply pep-8.

Also, `_pygamecanvas` is used like this in 21 activities, why change it to make this one different?